php-apache/usr/local/sbin/postfix.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# call "postfix stop" when exiting
+trap "{ echo Stopping Postfix; /usr/sbin/postfix stop; exit 0; }" EXIT
+
+source /etc/sysconfig/network
+
+/usr/libexec/postfix/aliasesdb
+/usr/libexec/postfix/chroot-update
+
+if [ -n "${POSTFIX_RELAY_USER}" ]; then
+  postmap /etc/postfix/sasl_passwd
+fi
+
```

Comment:
DNS resolution was a problem for us, so we had to copy /etc/resolv.conf to somewhere visible by postfix. Fix from @fratcliffe incoming